model/node_pool_config.go

```diff
@@ -13,9 +13,10 @@ type NodePoolConfig struct {
 	InstanceType       string            `yaml:"instanceType,omitempty"`
 	ManagedIamRoleName string            `yaml:"managedIamRoleName,omitempty"`
 	RootVolume         `yaml:",inline"`
-	SpotPrice          string   `yaml:"spotPrice,omitempty"`
-	SecurityGroupIds   []string `yaml:"securityGroupIds,omitempty"`
-	Tenancy            string   `yaml:"tenancy,omitempty"`
+	SpotPrice          string      `yaml:"spotPrice,omitempty"`
+	SecurityGroupIds   []string    `yaml:"securityGroupIds,omitempty"`
+	Tenancy            string      `yaml:"tenancy,omitempty"`
+	EbsVolumes         []EbsVolume `yaml:"ebsVolumes,omitempty"`
```

Comment:
Should be changed to `VolumeMounts` if we go ahead with https://github.com/coreos/kube-aws/pull/342/files#r102595612